site/specs-temp/numbers.json

```diff
@@ -17123,21 +17123,21 @@
                   },
                   "orderIsCanceled": {
                     "summary": "Order is canceled",
-                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<TollFreePortingValidationResponses>\n<TollFreePortingValidationResponse>\n  <TollFreePortingValidation>\n    <CustomerOrderId>MyOptionalOrderId</CustomerOrderId>\n    <ProcessingStatus>CANCELLED</ProcessingStatus>\n    <AccountId>9900572</AccountId>\n    <CreatedByUser>jgilmore</CreatedByUser>\n    <OrderCreateDate>2020-08-20T14:51:58.695Z</OrderCreateDate>\n    <OrderId>e2b029cf-1cfa-4285-a875-80e8fd951208</OrderId>\n    <TollFreeNumberList>\n      <TollFreeNumber>8442948899</TollFreeNumber>\n        <TollFreeNumber>8774024485</TollFreeNumber>\n      </TollFreeNumberList>\n  </TollFreePortingValidation>\n</TollFreePortingValidationResponse>\n</TollFreePortingValidationResponses>"
+                    "value": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?>\n<TollFreePortingValidationResponses>\n  <TollFreePortingValidationResponse>\n    <TollFreePortingValidation>\n      <CustomerOrderId>MyOptionalOrderId</CustomerOrderId>\n      <ProcessingStatus>CANCELLED</ProcessingStatus>\n      <AccountId>9900572</AccountId>\n      <CreatedByUser>jgilmore</CreatedByUser>\n      <OrderCreateDate>2020-08-20T14:51:58.695Z</OrderCreateDate>\n      <OrderId>e2b029cf-1cfa-4285-a875-80e8fd951208</OrderId>\n      <TollFreeNumberList>\n        <TollFreeNumber>8442948899</TollFreeNumber>\n          <TollFreeNumber>8774024485</TollFreeNumber>\n        </TollFreeNumberList>\n    </TollFreePortingValidation>\n  </TollFreePortingValidationResponse>\n</TollFreePortingValidationResponses>"
```

Comment:
Some of the indentation on the TN list looks a bit wonky in the cancelled case.
<img width="472" alt="image" src="https://user-images.githubusercontent.com/76067287/184002389-2f1a5d90-c0a0-478e-b53b-0cd1b1b33811.png">

Reply:
Thanks! I've been hunting these issues down to fix as I come across them